portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -34,19 +34,30 @@
     
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-     //retrieves comment data from datastore
+     // Get the user input on how many comments to display.
+     // If user input for max comments is greater than all comments, all comments will display.
+     // If comments are empty, no comments will be displayed.
+     
+    Integer maxComments = getNumDisplayComments(request);
```

Comment:
You should generally prefer to use the primitive type when one's available, unless you really need to treat a value like an object for some reason.

```suggestion
    int maxComments = getNumDisplayComments(request);
```